src/Spark/Spark.csproj

```diff
@@ -138,29 +135,41 @@
     <Reference Include="System.IO.Compression.FileSystem" />
     <Reference Include="System.Net.Http, Version=4.1.1.2, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\..\packages\System.Net.Http.4.3.3\lib\net46\System.Net.Http.dll</HintPath>
+      <Private>True</Private>
```

Comment:
I think the `<Private>` nodes should only be applied once

Reply:
Agree, removed the duplicated nodes